lib/scores.js

```diff
@@ -0,0 +1,114 @@
+const Async = require('async');
+const level = require('../lib/level.js');
```

Comment:
Cool. ok. as long as the team are _aware_ that we either need to implement "_sync_" between the node.js servers (_if they use LevelDB_) or use ElastiCache to store the data, I'm happy with either. 👍